test/fixture/no_msvc/no_msvcs_sconstruct_tools.py

```diff
@@ -0,0 +1,14 @@
+import SCons
+import SCons.Tool.MSCommon
+
+def DummyVsWhere(msvc_version, env):
+    # not testing versions with vswhere, so return none
+    return None
+
+for key in SCons.Tool.MSCommon.vc._VCVER_TO_PRODUCT_DIR:
+    SCons.Tool.MSCommon.vc._VCVER_TO_PRODUCT_DIR[key]=[(SCons.Util.HKEY_LOCAL_MACHINE, r'')]
+
+SCons.Tool.MSCommon.vc.find_vc_pdir_vswhere = DummyVsWhere
+
+env = SCons.Environment.Environment(tools=['MYIGNOREDEFAULTMSVCTOOL'])
```

Comment:
does this work? shouldn't it be `tools=['myignoredefaultmsvctool']`

Reply:
I was confident that it works.  Now it doesn't seem like it should.  Although the intention was to have a tool list that does not contain an msvc tool which is probably satisfied.  Will investigate and likely change.